subsys/net/lib/nrf_cloud/src/nrf_cloud_info.c

```diff
@@ -140,25 +141,30 @@ int nrf_cloud_print_details(void)
 	LOG_INF("Sec tag:   %d", nrf_cloud_sec_tag_get());
 	LOG_INF("Host name: %s", host_name);
 
-	/* Tenant will not be known unless device is connected to cloud with MQTT. */
+#endif /* CONFIG_NRF_CLOUD_VERBOSE_DETAILS */
+
+	return err;
+#else
+	return 0;
+#endif /* CONFIG_NRF_CLOUD_PRINT_DETAILS */
+}
+
+int nrf_cloud_print_cloud_details(void)
+{
+	int err = 0;
+#if defined(CONFIG_NRF_CLOUD_PRINT_DETAILS) && defined(CONFIG_NRF_CLOUD_VERBOSE_DETAILS)
```

Comment:
```suggestion
#if defined(CONFIG_NRF_CLOUD_VERBOSE_DETAILS)
```
if kconfig `depends on` suggestion is added